src/test/scala/za/co/absa/commons/io/TempFileSpec.scala

```diff
@@ -40,4 +41,20 @@ class TempFileSpec extends AnyFlatSpec with Matchers {
     name2 should endWith("bar")
     name3 should (startWith("foo") and endWith("bar"))
   }
+
+  it should "return tempFile as string" in {
```

Comment:
ditto

Reply:
Fixed in c6675808ba03a8efcc7111030996fdde162b626a.